mollie/api/objects/order_line.py

```diff
@@ -107,16 +118,6 @@ def product_url(self):
     def created_at(self):
         return self._get_property('createdAt')
 
-    def cancel(self):
```

Comment:
This method can still exist, it should call the DELETE with only itself as the argument.

Reply:
Correction: we don't have any cancel methods anymore on result objects, so this one should go too.